src/Agent.Worker/Release/Artifacts/JenkinsArtifact.cs

```diff
@@ -104,6 +112,10 @@ public IArtifactDetails GetArtifactDetails(IExecutionContext context, AgentArtif
 
             var allFieldsPresents = artifactDetails.TryGetValue("RelativePath", out relativePath)
                                     && artifactDetails.TryGetValue("JobName", out jobName);
+
+            bool _acceptUntrusted = jenkinsEndpoint.Data.ContainsKey("acceptUntrustedCerts") &&
```

Comment:
I believe this would be the case for Jenkins endpoints created before the "UntrustedCertificate" support was added. Would be good to validate with an endpoint that doesn't have this setting as part of it.